spec/controllers/admin/events_controller_spec.rb

```diff
@@ -0,0 +1,26 @@
+require 'spec_helper'
+
+describe Admin::EventsController do
+  let(:conference) { create(:conference) }
+  let(:organizer_role) { Role.find_by(name: 'organizer', resource: conference) }
+  let(:organizer) { create(:user, role_ids: organizer_role.id) }
+  let(:event_without_commercial) { create(:event, program: conference.program) }
+  let(:event_with_commercial) { create(:event, program: conference.program) }
+  let(:event_commercial) { create(:event_commercial, commercialable: event_with_commercial, url: 'https://www.youtube.com/watch?v=M9bq_alk-sw') }
+
+  with_versioning do
+    describe 'GET #show' do
+      before :each do
+        sign_in(organizer)
+        event_without_commercial
```

Comment:
For the 3 events you want created, don't call them here, I think we are just adding extra code for no reason.